wiki/osu!dev_Discord_server/pt-br.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - development
+---
+
+# Servidor do Discord osu!dev 
+
+**osu!dev** é o servidor oficial do [Discord](https://discordapp.com "Discord's website") para trabalhar com o código aberto do osu! e projetos da comunidade. Este é o local principal para contribuir com o desenvolvimento do osu! e entrar em contato com o [osu!team](/wiki/People/The_Team).
```

Comment:
```suggestion
O **osu!dev** é o servidor oficial do [Discord](https://discordapp.com "Site do Discord") para trabalhar com o código aberto do osu! e projetos da comunidade. Este é o local principal para contribuir com o desenvolvimento do osu! e entrar em contato com o [osu!team](/wiki/People/The_Team).
```
`O` in the beginning is just a suggestion. The latter must be translated since it's a tip balloon when you hover over the link.